server/app/mutations/grid_services/common.rb

```diff
@@ -120,43 +119,56 @@ def build_service_volumes(existing_volumes, grid, stack)
       service_volumes
     end
 
-    # @param [Grid] grid
-    # @param [Stack] stack
-    # @param [Array<Hash>] links
-    def validate_links(grid, stack, links)
-      links.each do |link|
-        link[:name] = "#{stack.name}/#{link[:name]}" unless link[:name].include?('/')
-        linked_stack, service_name = parse_link(grid, link)
+    def validate_links
+      validate_each :links do |link|
```

Comment:
It was part of the `Mutations::Command` monkey-patch, moved to an included  helper module.